packages/ethereum/package.json

```diff
@@ -14,9 +14,9 @@
     "build:sol:types": "echo \"Generating smart contract typings\" && TS_NODE_PROJECT=./tsconfig.hardhat.json hardhat typechain",
     "prepack": "rm -rf deployments/hardhat-localhost deployments/hardhat-localhost2",
     "fund": "TS_NODE_PROJECT=./tsconfig.hardhat.json hardhat fund",
-    "faucet": "TS_NODE_PROJECT=./tsconfig.hardhat.json hardhat faucet",
+    "faucet": "TS_NODE_PROJECT=./tsconfig.hardhat.json HOPR_ENVIRONMENT_ID=hardhat-localhost hardhat faucet --use-local-identities --identity-directory /tmp --identity-prefix local --password local",
     "accounts": "TS_NODE_PROJECT=./tsconfig.hardhat.json hardhat accounts",
-    "network": "TS_NODE_PROJECT=./tsconfig.hardhat.json DEVELOPMENT=true hardhat node",
+    "network": "TS_NODE_PROJECT=./tsconfig.hardhat.json HOPR_ENVIRONMENT_ID=hardhat-localhost hardhat node",
```

Comment:
README.md might need update, if this env variable does not need to be specified anymore when running `network`.

Reply:
Checked, development section assumes already the new usage, so no update required. Conversely, the current README.md might lead to problems.